src/main/java/mainpack/MainRun.java

```diff
@@ -52,6 +58,30 @@ else if(choice > 1 && choice < 6){
                                 database.getList().get(index).getSurname());
                     }
                     else if (choice == 3) {
+                        System.out.println("Choose the client");
+                        handler.printClients(database);
+                        index = scanner.nextInt();
+                        if(database.getList().get(index).getAgreementList().isEmpty()){
+                            System.out.println("There are no agreements for this client");
+                        } else {
+                            handler.printAgreements(database,index);
+                            System.out.println("Choose the agreement");
+                            index2 = scanner.nextInt();
+                            menu.subjectMenu();
+                            int choiceSub = scanner.nextInt();
+                            AbstractSubject tmp;
+                            if(choiceSub == 1){
+                                tmp = new House();
+                            } else {
+                                tmp = new Apartment();
+                            }
```

Comment:
So if user choose variant that is not 1 (e.g. 2, 3, -333, and so on), then should be created Apartment subject?